README_v2.md

```diff
@@ -0,0 +1,176 @@
+    FIXME the labels in the README do NOT match what is in the bin/solr start -help, and are better!
+    cloud:         SolrCloud example
+    techproducts:  Comprehensive example illustrating many of Solr's core capabilities
+    schemaless:    Schema-less example (schema is inferred from data during indexing)
+    films:         Example of starting with _default configset and adding explicit fields dynamically    
+```
+
+For instance, if you want to run the SolrCloud example, enter:
+
+```
+  bin/solr -e cloud
+```
+
+### Running Solr in Docker
+
+You can run Solr in Docker via the [official image](https://hub.docker.com/_/solr).
+
+To run Solr in a container and expose the Solr port, run:
+
+`docker run -p 8983:8983 solr`
+
+In order to start Solr in cloud mode, run the following.
+
+`docker run -p 8983:8983 solr solr-fg -c`
+
+For documentation on using the official docker builds, please refer to the [DockerHub page](https://hub.docker.com/_/solr).  
+Up to date documentation for running locally built images of this branch can be found in the [local reference guide](solr/solr-ref-guide/src/running-solr-in-docker.adoc).
+
+There is also a gradle task for building custom Solr images from your local checkout.
+These local images are built identically to the official image except for retrieving the Solr artifacts locally instead of from the official release.
+This can be useful for testing out local changes as well as creating custom images for yourself or your organization.
+The task will output the image name to use at the end of the build.
+
+`./gradlew docker`
+
+For more info on building an image, run:
+
+`./gradlew helpDocker`
+
+### Running Solr on Kubernetes
+
+Solr has official support for running on Kubernetes, in the official Docker image.
+Please refer to the [Solr Operator](https://solr.apache.org/operator) home for details, tutorials and instructions.
+
+## Building Solr from Source
+FIXME --> is this where we get JDK from??   What about OpenJDK?
+Download the Java 11 JDK (Java Development Kit) or later from https://jdk.java.net/
+You will need the JDK installed, and the $JAVA_HOME/bin (Windows: %JAVA_HOME%\bin)
+folder included on your command path. To test this, issue a "java -version" command
+from your shell (command prompt) and verify that the Java version is 11 or later.
+
+Download the Apache Solr distribution, from http://solr.apache.org/.  FIXME link??
+Unzip the distribution to a folder of your choice, e.g. C:\solr or ~/solr
+Alternately, you can obtain a copy of the latest Apache Solr source code
+directly from the Git repository:
+
+<https://solr.apache.org/community.html#version-control>
+
+As of 9.0, Solr uses [Gradle](https://gradle.org/) as the build
+system.  Navigate to the root of your source tree folder and issue the `./gradlew tasks`
+command to see the available options for building, testing, and packaging Solr.
+
+`./gradlew assemble` will create a Solr executable.
```

Comment:
Perhaps highlight the `./gradlew dev` task instead, as it it creates a build on a stable path and is made for quick developer cycles. Then we won't need to change the README for every release either, replacing the `solr-9.0.0-SNAPSHOT` string :)

Reply:
Good suggestion.   I added a mention of `assemble` later on.